src/main/java/core/Game.java

```diff
@@ -799,119 +790,6 @@ public String toString() {
         return gameType.toString();
     }
 
-    public void setupVideoRecording(String filename, String formatname,
```

Comment:
I'd prefer to leave the video recording in.

Reply:
Looks like an accidental merge issue - I'll get it resurrected.